database.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
+s,banana,100
+p,banana,13
+r,apple,10
+p,apple,20
+p,banana,5
+s,banana,50
```

Comment:
Place the input and output files into the src/main/resources folder -  [https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop](url)